app/models/consumer_app.rb

```diff
@@ -0,0 +1,60 @@
+class ConsumerApp < ApplicationRecord
+  resourcify
+
+  FOREM_BUNDLE = "com.forem.app".freeze
+  SUPPORTED_PLATFORMS = [Device::ANDROID, Device::IOS].freeze
+  FOREM_APP_PLATFORMS = [Device::IOS].freeze
+
+  validates :app_bundle, presence: true
+  validates :platform, presence: true
```

Comment:
this is missing the unique validation mirroring the unique constraint, without it the user will be shown the PostgreSQL internal error

Reply:
Just FYI (since this comment doesn't appear as "outdated"): I added this validation in the line after this one ✅